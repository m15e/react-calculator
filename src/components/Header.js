import React from 'react';

const Header = () => (
  <div className="header-container">
    <ul>
      <li><a href="/">Math Magicians</a></li>
      <li><a href="/">Home</a></li>
      <li><a href="/calculator">Calculator</a></li>
      <li><a href="/quote">Quote</a></li>
    </ul>
  </div>
);

export default Header;
