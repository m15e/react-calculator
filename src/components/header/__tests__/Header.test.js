import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

it('Header component matches snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
