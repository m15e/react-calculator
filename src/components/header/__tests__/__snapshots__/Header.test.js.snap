// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<div
  className="header-container"
>
  <ul>
    <li>
      <a
        href="/"
      >
        Math Magicians
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
